test/SlowTests/Issues/RavenDB_21681.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FastTests.Utils;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.Expiration;
+using SlowTests.Core.Utils.Entities;
+using Sparrow;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_21681 : ReplicationTestBase
+    {
+        public RavenDB_21681(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenTheory(RavenTestCategory.Cluster | RavenTestCategory.Replication | RavenTestCategory.TimeSeries | RavenTestCategory.ExpirationRefresh)]
+        [RavenData(DatabaseMode = RavenDatabaseMode.All)]
+        public async Task DeleteExpiredDocumentWithBigTimeSeriesShouldNotCauseReplicationToBreak(Options options)
+        {
+            var databaseName = GetDatabaseName();
+            var (nodes, leader) = await CreateRaftCluster(2);
+            var (_, servers) = await CreateDatabaseInClusterForMode(databaseName, 2, (nodes, leader), options.DatabaseMode);
+
+            using (var store = new DocumentStore { Database = databaseName, Urls = new[] { leader.WebUrl } }.Initialize())
+            {
+                var user = new User { Name = "Shiran" };
+
+                var expiry = DateTime.Now.AddYears(-1).ToUniversalTime();
+                using (var session = store.OpenAsyncSession())
+                {
+                    await session.StoreAsync(user);
+                    var metadata = session.Advanced.GetMetadataFor(user);
+                    metadata[Constants.Documents.Metadata.Expires] = expiry.ToString(DefaultFormat.DateTimeFormatsToRead[0]);
+                    await session.SaveChangesAsync();
+                }
+
+                using (var session = store.OpenAsyncSession())
+                {
+                    var ts = session.TimeSeriesFor(user.Id, "heartbeat");
+                    for (int i = 0; i < 4_000_000; i++)
```

Comment:
wow :-)
how long does this test takes to pass? and what is the memory consumption of this..
either reduce this, or move to stress tests (or both)